pkg/apis/troubleshoot/v1beta2/analyzer_shared.go

```diff
@@ -12,7 +12,7 @@ type ClusterVersion struct {
 type StorageClass struct {
 	AnalyzeMeta      `json:",inline" yaml:",inline"`
 	Outcomes         []*Outcome `json:"outcomes" yaml:"outcomes"`
-	StorageClassName string     `json:"storageClassName" yaml:"storageClassName"`
+	StorageClassName string     `json:"storageClassName,omitempty" yaml:"storageClassName,omitempty"`
```

Comment:
@xavpaice I'm guessing by the code path that this only applies for api version `troubleshoot.replicated.com/v1beta2`, but https://github.com/replicated-collab/cased-kots/issues/25 is using `v1beta1`. Does this still apply in that case?

Reply:
thanks, I'm not sure we should be making any changes to v1beta1, isn't it there because of history and needing to be (eventually) deprecated?

Reply:
I don't see a big problem with such a minor update - ran it anyway.

Reply:
Really good point, I honestly don't know.